frontend/src/scenes/persons/Person.tsx

```diff
@@ -201,7 +207,25 @@ export function Person(): JSX.Element | null {
                         tab={<span data-attr="persons-related-flags-tab">Feature flags</span>}
                         key={PersonsTabType.FEATURE_FLAGS}
                     >
-                        <RelatedFeatureFlags distinctId={person.distinct_ids[0]} />
+                        <div className="flex space-x-4 items-center mb-2">
+                            <div>
+                                Choose ID:
+                                <Tooltip title="Feature flags can have different values based on the persons IDs. Turn on persistence in feature flag settings if you'd like these to be constant always.">
+                                    <InfoCircleOutlined style={{ marginLeft: 6, marginRight: 0 }} />
+                                </Tooltip>
+                            </div>
+                            <LemonSelect
+                                value={person.distinct_ids[0]}
```

Comment:
The force index here is 😬 but i guess it should never be empty

Reply:
Yeah, the person can't exist without atleast one distinct ID.